lib/internal/buffer_old.js

```diff
@@ -11,6 +11,7 @@ const kMaxLength = smalloc.kMaxLength;
 exports.Buffer = Buffer;
 exports.SlowBuffer = SlowBuffer;
 exports.INSPECT_MAX_BYTES = 50;
+exports.kMaxLength = binding.kMaxLength;
```

Comment:
The size limit of ArrayBuffer no longer exists, but number of array indices on a typed array are still constrained. I believe his comment originated from my original belief that typed arrays had no upper limit as well. That was my bad.